client/js/main.js

```diff
@@ -58,6 +58,9 @@ function getValidDropTargets(widget) {
         break;
     }
 
+    if (getComputedStyle(t.domElement).getPropertyValue('--foreign') == 'true')
```

Comment:
Is this check faster than refactoring this into a function and call it (and only `get('owner')` once):

```Javascript
    if(Array.isArray(this.get('owner')) && this.get('owner').indexOf(playerName) == -1)
      className += ' foreign';
    if(typeof this.get('owner') == 'string' && this.get('owner') != playerName)
      className += ' foreign';
```

Reply:
I'm using the css inheritance to automatically apply this to descendants within foreign holders.